idl/forward_request.idl.hh

```diff
@@ -1,9 +1,12 @@
 namespace query {
 struct forward_request {
-    enum class reduction_type : uint8_t {
-        count,
+    struct count {};
+    struct uda {
+        sstring keyspace;
+        sstring uda_name;
+        std::vector<sstring> column_names;
     };
```

Comment:
You can't change IDL in a non-backward-compatible manner, it breaks upgrades.

Reply:
We discussed this one briefly, and since distributed COUNT(*) didn't make it to any release so far, I think it's much better to break now, since nobody is going to upgrade from one unofficial commit to another, than to have a weird optional<variant> at the end of this request, which renders the original enum rather redundant. Similarly, this series also piggy-backs on count(*)'s cluster feature for the same reason - to avoid churn since we're not part of any official release.

Reply:
That depends on when 5.1 will be branched and when this series makes it in.

Reply:
Now IDL is changed properly, only one new field added at the end, optional and annotated with `[[version 5.1]]`